Build/AzureDevOps/azure-pipelines.yml

```diff
@@ -0,0 +1,87 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
+
+jobs:
+  - job: Windows
+
+    pool:
+      vmImage: 'VS2017-Win2016'
+
+    variables:
```

Comment:
We should be running this for win32/x64/arm, debug/release - do you know if YAML supports comma-separated variables yet?

Reply:
Agreed.  This is just a first step to getting something working.  I'm also looking at adding Android too.  This build also doesn't generate Nuget packages so it isn't yet a replacement for the other builds.